src/components/Notes.js

```diff
@@ -7,52 +7,58 @@ import { grey } from "../colors"
 import Fold from "../primitives/Fold"
 import moment from "moment"
 
+// Reakit import
+import theme from "reakit-theme-default";
+// Confusing part is that
+//import { Button, Provider } from "reakit";
```

Comment:
Can you clean up these comments?

Do you have any questions related to this?

Reply:
Ahh, I am struggling with the grid set up that we are using, but wanted to use some existing Reakit elements and their theming. I'll make an issue to style the Notes page down the road if we feel it is necessary.